homeassistant/components/systemair/sensor.py

```diff
@@ -0,0 +1,505 @@
+            'Inside Humidity',
+            '%',
+            'mdi:water-percent',
+            SENSOR_CURRENT_HUMIDITY
+        ],
+
+        ATTR_FILTER_TIME: [
+            'Filter Replacement',
+            'days',
+            'mdi:alarm',
+            SENSOR_FILTER_TIME
+        ],
+
+        ATTR_CURRENT_FAN_MODE: [
+            'Fan Mode',
+            '',
+            'mdi:fan',
+            SENSOR_CURRENT_FAN_MODE
+        ],
+
+        ATTR_CURRENT_OPERATION: [
+            'Operation Mode',
+            '',
+            'mdi:water-percent',
+            SENSOR_CURRENT_OPERATION
+        ],
+
+        ATTR_USER_LOCK: [
+            'User Lock',
+            '',
+            'mdi:account-alert',
+            SENSOR_USER_LOCK
+        ],
+
+        ATTR_FUNCTION_COOLING: [
+            'Cooling',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_COOLING
+        ],
+
+        ATTR_FUNCTION_VACUUM_CLEANER: [
+            'Vacuum Cleaner',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_VACUUM_CLEANER
+        ],
+        ATTR_FUNCTION_FREE_COOLING: [
+            'Free Cooling',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_FREE_COOLING
+        ],
+        ATTR_FUNCTION_HEATING: [
+            'Heating',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_HEATING
+        ],
+        ATTR_FUNCTION_DEFROSTING: [
+            'Defrosting',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_DEFROSTING
+        ],
+        ATTR_FUNCTION_HEAT_RECOVERY: [
+            'Heat Recovery',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_HEAT_RECOVERY
+        ],
+        ATTR_FUNCTION_COOLING_RECOVERY: [
+            'Cooling Recovery',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_COOLING_RECOVERY
+        ],
+        ATTR_FUNCTION_MOISTURE_TRANSFER: [
+            'Moisture Transfer',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_MOISTURE_TRANSFER
+        ],
+        ATTR_FUNCTION_SECONDARY_AIR: [
+            'Secondary Air',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_SECONDARY_AIR
+        ],
+        ATTR_FUNCTION_COOKER_HOOD: [
+            'Cooker Hood',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_COOKER_HOOD
+        ],
+        ATTR_FUNCTION_HEATER_COOLDOWN: [
+            'Heater Cooldown',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_HEATER_COOLDOWN
+        ],
+        ATTR_FUNCTION_SERVICE_USER_LOCK: [
+            'Service User Lock',
+            '',
+            'mdi:bell',
+            SENSOR_FUNCTION_SERVICE_USER_LOCK
+        ],
+
+
+        ATTR_ALARM_FROST_PROTECTION: [
+            'Alarm Frost Protection',
+            '',
+            'mdi:alert',
+            SENSOR_ALARM_FROST_PROTECTION
+        ],
+        ATTR_ALARM_FROST_PROTECTION_SENSOR: [
```

Comment:
What kind of state value does this return? If it's a binary value, this should move to a binary sensor platform. Same goes for all the alarm types.